plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/editor/signature/ParamterInfoImpl.java

```diff
@@ -30,6 +30,10 @@ public String getLabel() {
 
   @Override
   public Optional<String> getDocumentation() {
-    return Optional.fromNullable(dto.getDocumentation());
+    if (dto.getDocumentation().isLeft()) {
+      return Optional.fromNullable(dto.getDocumentation().getLeft());
+    }
+    // TODO: need to find solution for using markdown content @see org.eclipse.lsp4j.MarkupContent
+    return Optional.fromNullable(dto.getDocumentation().getRight().getValue());
```

Comment:
Can dto.getDocumentation() not return null? According to https://microsoft.github.io/language-server-protocol/specification#textDocument_signatureHelp the documentation field is optional, so could be null.